libhoney.go

```diff
@@ -204,7 +197,6 @@ func Init(config Config) error {
 	blockOnResponses = config.BlockOnResponse
 
 	// reset the global transmission
-	tx.Stop()
```

Comment:
yep - just removing this might leak if we removed this and users call `Init()` a second/multiple times, but that's fairly nonstandard/discouraged behavior anyway =\